### DISCUSSION TITLE
Secondary development

### DISCUSSION THREAD

**Discussion by 2449247266wqq** [dbeaver/dbeaver#17104] [DBeaver Development]
I needed to re-develop DBeaver to support a new database connection, but after downloading the open source code online, there were a lot of plugins involved in the code, and the code framework was not clear, so it was difficult to modify the code. At the same time, I want to debug dbeaver in eclipse so that I can see the results after modifying the code. Please guide you on how to debug the dbeaver project in eclipse, while kneeling down for an explanation of the dbeaver code structure.Thinks

**Comment by LonwoLonwo** [comment]
Hello. We have only this page about development in Eclipse: https://github.com/dbeaver/dbeaver/wiki/Develop-in-Eclipse
